modules/serverless-init-containers-apps.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * /serverless/develop/serverless-applications.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-init-containers-apps_{context}"]
+= Configuring init containers
+
+link:https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[Init containers] are specialized containers that are run before application containers in a pod. They are generally used to implement initialization logic for an application, which may include running setup scripts or downloading required configurations.
+
+[NOTE]
+====
+Init containers may cause longer application start-up times and should be used with caution for serverless applications, which are expected to scale up and down frequently.
+====
+
+Multiple init containers are supported in a single Knative service spec. Knative provides a default, configurable naming template if a template name is not provided. The init containers template can be set by adding an appropriate value in a Knative `Service` object spec.
+
+.Prerequisites
+
+* {ServerlessOperatorName} and Knative Serving are installed on your cluster.
+
+* Before you can use init containers for Knative services, an administrator must add the `kubernetes.podspec-init-containers` flag to the `KnativeServing` custom resource (CR). See the {ServerlessProductName} "Global configuration" documentation for more information.
+
+.Procedure
+
+* Add the `initContainers` spec to a Knative `Service` object:
++
+.Example service spec
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+...
+spec:
+  template:
+    spec:
```

Comment:
I would think these specs would be OK since they're part of the YAML code.

Reply:
All the "spec" mentions in these docs refer to this specific part of the YAML, which is always just called spec for kubernetes objects AFAIK, especially for Knative